apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/supervisor/msg/SvBusError.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright © 2020 Apollo Foundation
+ */
+package com.apollocurrency.aplwallet.apl.util.supervisor.msg;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Error message body on the bus
+ *
+ * @author alukin@gmail.com
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SvBusError {
+
+    public Integer error_code;
```

Comment:
I assume that the "errorCode" field must be mandatory with the primitive type int. 
We can use @JsonProperty("error_code") and  
@JsonAlias to change the property name.

Reply:
Camel case is good practice but not the dogma for Json variables.